buildpack.toml

```diff
@@ -29,26 +29,13 @@ api = "0.8"
     uri = "https://artifacts.paketo.io/icu/icu_75.1_linux_jammy_fc477acf.tgz"
     version = "75.1"
 
-  [[metadata.dependencies]]
-    checksum = "sha256:f39cfa7e15735a139470969a32a034dba2f98de0017dd753e98c0ac4eecd33d4"
-    cpe = "cpe:2.3:a:icu-project:international_components_for_unicode:75.1:*:*:*:*:c\\/c\\+\\+:*:*"
-    id = "icu"
-    licenses = ["BSD-2-Clause", "BSD-3-Clause", "ICU", "Unicode-TOU"]
-    name = "ICU"
-    purl = "pkg:generic/icu@75.1?checksum=70ea842f0d5f1f6c6b65696ac71d96848c4873f4d794bebc40fd87af2ad4ef064c61a786bf7bc430ce4713ec6deabb8cc1a8cc0212eab148cee2d498a3683e45&download_url=https://github.com/unicode-org/icu/releases/download/release-75-1/icu4c-75_1-src.tgz"
-    source = "https://github.com/unicode-org/icu/releases/download/release-75-1/icu4c-75_1-src.tgz"
-    source-checksum = "sha512:70ea842f0d5f1f6c6b65696ac71d96848c4873f4d794bebc40fd87af2ad4ef064c61a786bf7bc430ce4713ec6deabb8cc1a8cc0212eab148cee2d498a3683e45"
-    stacks = ["io.buildpacks.stacks.bionic"]
-    uri = "https://artifacts.paketo.io/icu/icu_75.1_linux_bionic_f39cfa7e.tgz"
-    version = "75.1"
-
   [[metadata.dependency-constraints]]
     constraint = "*"
     id = "icu"
     patches = 1
 
 [[stacks]]
-  id = "io.buildpacks.stacks.bionic"
+  id = "io.buildpacks.stacks.jammy"
 
 [[stacks]]
-  id = "io.buildpacks.stacks.jammy"
+  id = "io.buildpacks.stacks.noble"
```

Comment:
If I understand correctly, this being here should trigger automation to create artifact for noble and a PR to add the dependency in